app/uk/gov/hmrc/hecstubs/controllers/CitizenDetailsController.scala

```diff
@@ -26,12 +26,19 @@ import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 @Singleton
 class CitizenDetailsController @Inject() (cc: ControllerComponents) extends BackendController(cc) with Logging {
 
+  /**
+    * Stub for fetching individual details from the Citizens Details Service.
+    * If NINO starts with "NOUTR" then UTR returned in empty.
+    * @param ninoNoSuffix The NINO suffix for the individual
+    * @return The user's details
+    */
   def citizenDetails(ninoNoSuffix: String): Action[AnyContent] = Action { _ =>
+    val maybeUtr     = if (ninoNoSuffix.startsWith("NOUTR")) None else Some("1234567895")
```

Comment:
the form of a NINO without a suffix is two letters followed by 6 digits, like  `AA123456` - the filtering logic needs to be consistent with this, maybe something like `ifNinoNoSuffix.startsWith("NS")` or something? The first two letters are actually restricted as well - the rules are spelt out here:
https://github.com/hmrc/domain/blob/master/src/main/scala/uk/gov/hmrc/domain/Nino.scala#L49

Reply:
There's no validation on the incoming NINO, so I imagined we can be flexible with it? Do you think we need to add validation?

Reply:
we probably don't need to go as far as doing a full validation of the NINO, but when using the auth login wizard we'd only have the ability to enter in valid looking NINO's cos they put validation on that auth login wizard page in the NINO, so `ninoNoSuffix.startsWith("NOUTR")` would never be `true`. We just need to make sure that the condition can be triggered.

Reply:
ah got you, updated now 👍